app/styles/main.scss

```diff
@@ -42,6 +47,23 @@ $state-pending: $gray-light;
 $state-processing: #7facc5;
 $box-shadow-color: rgba(0, 0, 0, .075);
 
+
+// Colors from Bert's design:
+$slightly-white: #f1f1ee;
+
+$slightly-gray: #e5f3f8;
+$scenario-item-background: #e5f3f8;
+$scenario-selected-border: #008bbf;
+$scenario-selected-background: #008bbf;
+$search-domain-selected: #008bbf;
+$panel-gray: #dfdfda;
+$dark-gray: #cccccc;
+$slider-handle-border: #999999;
+$progress-bar-color: #cc4444;
+$badge-color-unselected: #989a9a;
+$button-color-nav-bar: #999999;
+$button-color-nav-bar-default: #008bbf;
```

Comment:
Very good! I would expect naming to be consistent though: either name them all as colours (e.g. `$red`, `$blue`, `$dark-blue`) or name them all after their purpose (e.g. `$background`, `$primary-1`, `$primary-2`, `$alert`, `$success`).

Do not name them after component(-classe)s: that defeats the purpose (as you would have to assign a component with the color of another component).